code/modules/holodeck/computer.dm

```diff
@@ -414,5 +421,14 @@ GLOBAL_LIST_INIT(typecache_holodeck_linked_floorcheck_ok, typecacheof(list(/turf
 	emergency_shutdown()
 	return ..()
 
+/obj/machinery/computer/holodeck/small
+	mapped_start_area = /area/holodeck/small
+	program_type = /datum/map_template/holodeck/small
+	offline_program = "Small - Offline"
+
+/obj/machinery/computer/holodeck/small/LateInitialize()
+  linked = /area/holodeck/small
```

Comment:
This line doesn't do anything since the first thing the parent call does is overwrite the value of linked.